apps/website/publisher.config.ts

```diff
@@ -36,4 +36,15 @@ export default defineConfig({
   },
   databaseUrl: 'persisted-store/publisher.sqlite',
   publisherPort: isLagoon ? 3000 : 8000,
+  oAuth2: {
```

Comment:
Yes, I was also thinking that it could be more simple for project specific things.

- For this `/admin/config/people/simple_oauth` most of the config is done, we just need to create the certificates per project
- For `/admin/config/services/consumer` that's content entities / has to be done per environment
- Environment variables could also be automated

I will check what could already be done with the init script.

Reply:
Oh, btw, the code blocks in the init script are marked as `ts`, but in fact it's JavaScript. TypeScript is not yet supported by `@amazeelabs/mzx` 🙈

Reply:
@Leksat i finally disabled it by default, everywhere. This might be better DX and so we can enable it only when requested + it won't complain when creating a new environment if something is not set.

As discussed: locally we are using `isLagoon` and skip in any case. I think there is no use case for that for now, and if we want to debug it locally it should be a task to be done in the `silverback-mono` Publisher package.

For Lagoon environments: we set `PUBLISHER_SKIP_AUTHENTICATION=true` by default and documentation was reviewed to make the setup clearer.